paper-input-error.html

```diff
@@ -65,20 +65,5 @@
     behaviors: [
       Polymer.PaperInputAddonBehavior
     ],
-
-    properties: {
-      /**
-       * True if the error is showing.
-       */
-      invalid: {
```

Comment:
If one binds to invalid$=[[item.inputInvalid]] , when loading an array  of objects to be spit through a dom-repeat to create a variety of inputs for a form, an empty value causes a styling reversion to a native input. 

True false and no bind do not have the effect.  The effect is not replicable outside the Dom repeat. 

Not sure if anyone else had the issue. 

If it's intentional it allows for a non-standard usage: typing the paper input to radio radio button and checkbox and using suffix plus no label float to crest check box and radio sets 

I don't mind the option, but do not believe it is intended.